index.js

```diff
@@ -13,8 +13,19 @@ module.exports = function (username, token) {
 			'user-agent': 'https://github.com/sindresorhus/gh-user'
 		}
 	}).then(function (result) {
-		delete result.body.gravatar_id;
-		delete result.body.bio;
-		return result.body;
+		var rateLimit = {'github-time': new Date(result.headers.date).getTime() / 1000};
```

Comment:
Why do we need to include `github-time`?

Also, it should be camelCased: `githubTime`. A better name might be: `serverTime`. ?